specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -1088,7 +1085,10 @@
                     "type": "object",
                     "properties": {
                       "results": {
-                        "$ref": "#/definitions/EntitiesResult"
+                        "$ref": "#/definitions/EntitiesResult",
```

Comment:
`EntitiesResult` already includes the stats. see https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L1517